onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -676,6 +676,24 @@ TEST_F(GraphTransformationTests, FuseCudaConvAddRelu) {
   ASSERT_TRUE(op_to_count["Add"] == 0);
   ASSERT_TRUE(op_to_count["Relu"] == 0);
 }
+
```

Comment:
Only suggestion would be a comment explaining what the bug is that this is trying to detect (or is it obvious to someone familiar with the optimizer?)

Reply:
Done.